source/configuration.rst

```diff
@@ -267,8 +266,17 @@ The only required configuration parameter is an execution target, which depends
       * ``Enterprise-My``, ``Enterprise-CA``, ``Enterprise-Root`` for machine enterprise store
       * ``GroupPolicy-My``, ``GroupPolicy-CA``, ``GroupPolicy-Root`` for machine group policy store
 
+    * Only on MacOSX, the user ('root' as a daemon) keychain will be used. You can use only the system keychain by using the ``system-ssl-ca`` key. (this option in not supported until v1.12)
+
+    By default, only ``CA`` and ``Root`` keystores for the machine's default, user ('LocalSystem' as a service), service, enterprise and 
+    group policy stores are exported on Windows.
+
+    On Unix/Linux, the agent uses the OpenSSL CA trust store (/etc/ssl/certs/ca-certificates.crt). (this option in not supported until v1.12)
+
```

Comment:
Only `system-ssl-ca` value should be documented with eventually an explanation on the context usage.

```suggestion
    * Only on MacOSX, the user ('root' as a daemon) keychain will be used.

      * You can force to use the system SSL CA keychain by using the ``system-ssl-ca`` key (Available since GLPI Agent v1.12)

    This can help if your certificate authority is not known by glpi-agent default SSL store based on `Mozilla::CA <https://metacpan.org/pod/Mozilla::CA>`_ but it is by system SSL CA.
```
Other sentences are no more on purpose.